MANAGEMENT_NETWORK.md

```diff
@@ -0,0 +1,318 @@
+network's provider and tenant networks. This attachment must be a bridgeable
+interface in the OpenvSwitch pod and must permit communication among other pods
+on the same node. The _bridge_ attachment type is the only type that
+supports this. While the _bridge_ attachment type does not enable connectivity
+across nodes on it's own, the vlan interface added to the bridge in the
+_NodeNetworkConfigurationPolicy_ above creates the necessary layer 2 link.
+
+```sh
+cat >> octavia-nad.yaml << EOF_CAT
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  labels:
+    osp/net: octavia
+  name: octavia
+  namespace: openstack
+spec:
+  config: |
+    {
+      "cniVersion": "0.3.1",
+      "name": "octavia",
+      "type": "bridge",
+      "bridge": "octbr",
+      "ipam": {
+        "type": "whereabouts",
+        "range": "172.23.0.0/24",
+        "range_start": "172.23.0.30",
+        "range_end": "172.23.0.70",
+        "routes": [
+           {
+             "dst": "172.24.0.0/16",
+             "gw" : "172.23.0.150"
+           }
+         ]
+      }
+    }
+EOF_CAT
+oc apply -n openstack -f octavia-nad.yaml
+```
+
+### Enabling octavia
+
+When enabling octavia, you need to configure the OVN controller to create a NIC
+mapping for the `octavia` network attachment as well as add it to the
+networkAttachments property for each Octavia service that controls amphorae.
+
+```sh
+# Note: please add 'octavia: octbr' to existing nicMappings if there are
+# existing values. Also the name 'controlplane' may be different for your
+# deployment.
+oc patch  -n openstack openstackcontrolplane controlplane --type=merge --patch '
+spec:
+  ovn:
+    template:
+      ovnController:
+        nicMappings:
+          octavia: octbr
+  octavia:
+    enabled: true
+    template:
+      octaviaHousekeeping:
+        networkAttachments:
+          - octavia
+      octaviaHealthManager:
+        networkAttachments:
+          - octavia
+      octaviaWorker:
+        networkAttachments:
+          - octavia
+'
+```
+
+When the operator is done deploying, the output of `oc get pods` should include
+lines similar to the following (the actual names will vary by suffix):
+
+```
+  octavia-api-5cf9bc78f7-4lmds                                    2/2     Running     0          42h
+  octavia-healthmanager-5g94j                                     1/1     Running     0          21h
+  octavia-housekeeping-5gtw8                                      1/1     Running     0          21h
+  octavia-image-upload-78b4b6c47c-xzdtl                           1/1     Running     0          35h
+  octavia-worker-pq55m                                            1/1     Running     0          21h
+```
+
+## The Octavia Neutron LB Management Network
+
+Once the octavia operator has finished deploying octavia, the details of the
+management network can be examined. The results of running `oc rsh openstack
```

Comment:
`openstackclient` (no whitespace in between)

Reply:
nice catch, thanks